rocAL/rocAL/CMakeLists.txt

```diff
@@ -261,6 +274,13 @@ if(${BUILD_ROCAL})
                 include/api/rocal_api_data_transfer.h
                 include/api/rocal_api_parameters.h
                 include/api/rocal_api_meta_data.h
+                include/pipeline/tensor.h
+                include/device/device_manager.h
+                include/device/device_data_transfer_code.h
+                include/device/device_code.h
+                include/pipeline/commons.h
+                include/pipeline/exception.h
+                include/pipeline/log.h
```

Comment:
The includes from L277 to L283 is part of this PR?

Reply:
Removed it

Reply:
removed